MANIFEST.in

```diff
@@ -1,7 +1,6 @@
-include *.rst
```

Comment:
I'm counting on #103 for this change :smile:

Reply:
Though, it will need to be replaced with ```*.md```...

Reply:
P.S. You have write access on this repo @ocefpaf. Why count on it, and not feel free to just merge 😜.

Reply:
Will do for simpler things, but it turns out I needed a review on this one after all.